### DISCUSSION TITLE
Installers for Win and Mac being readied

### DISCUSSION THREAD

**Discussion by leap71** [leap71/PicoGK#12] [Announcements]
Hi all, we have installers for Win and Mac now up and running. It's tested fairly well on both, but would be cool to see others give it a go.

@josefinelissner and I are working on updating the documentation for PicoGK and other modules, and then we can officially call it the 1.1 release.

On Windows, the installer also installs a test project in Documents/PicoGK/Example - after completed installation, as long as you have Visual Studio installed, you should be able to open the Playground project and run it without any additional input.

The installers can be found here: https://github.com/leap71/PicoGKInstaller/tree/main/Installers 

Let me know,
Lin

**Comment by Niels-IO** [comment]
Hi Lin,

I just tried to use the Mac Installer on a M1 MacBook and macOS wouldn't allow me to install it as the developer could not be verified. Any chance that there will be a verified installer available in the future?

Thanks,
Niels

**Reply by leap71** [reply]
Hi Niels — for now, you can allow the installer in the Mac security settings. We are considering signing the installers in the future, but keep in mind we are all doing this in our spare time.

Lin

**Reply by Niels-IO** [reply]
Hi Lin,

Thanks for the reply. I managed to build it myself using your very detailed instructions in the end. 

Just want to emphasize that I am very impressed by your library and that you made it open-source! Thanks a lot! 🙂

**Reply by LinKayser** [reply]
Awesome, show us what you create with it, if you want. We will open source more of our work in the future. 

Best,
Lin

**Comment by LinKayser** [comment]
You can now use the installers in https://github.com/leap71/PicoGK/releases/tag/PicoGK-v1.1.0
